runtime-2/fixtures/database.mp

```diff
@@ -0,0 +1,4 @@
+����happy�bool�health�integer��rep_1�rep_2��forest���choices­A Basic Event�This event has options���An Option without requirements����#Another option without requirements�����basic­A Basic Event�5This event has no options, just title and description����requirementsªConditions�This event has conditions���An Option���Resource���health�integer�Equals�100��TimeOfDay��Night�Equals��Event��choices�Depends��
```

Comment:
Do we need to store messagepack files? should we instead compile it in the test, decompile it and check that things are what we expect? this will be a maintenance nightmare, I stopped doing this after it became annoying.